src/pages/users/users.tsx

```diff
@@ -135,84 +131,97 @@ export const UsersPage: React.FC = () => {
         const item = row?.original;
         return (
           <div className="font-medium text-sm">
-            <RenderIf condition={item?.status?.toLowerCase() === "pending"}>
-              <p className="text-amber">{item?.status}</p>
+            <RenderIf condition={item?.status === 0}>
+              <p className="text-amber">Pending</p>
             </RenderIf>
-            <RenderIf condition={item?.status?.toLowerCase() === "approved"}>
-              <p className="text-green-base">{item?.status}</p>
+            <RenderIf condition={item?.status === 1}>
+              <p className="text-green-base">Active</p>
             </RenderIf>
-            <RenderIf condition={item?.status?.toLowerCase() === "suspended"}>
-              <p className="text-accent-primary">{item?.status}</p>
+            <RenderIf condition={item?.status === 2}>
+              <p className="text-accent-primary">Suspended</p>
             </RenderIf>
           </div>
```

Comment:
@AdejokeOgunyinka Create an enum for the status. See what I did on `pages/admin-accounts/index.tsx`